assembly/assembly-ide-war/pom.xml

```diff
@@ -347,6 +345,19 @@
                 <artifactId>exec-maven-plugin</artifactId>
                 <executions>
                     <execution>
+                        <id>gwt-xml</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>java</goal>
+                        </goals>
+                        <configuration>
+                            <mainClass>org.eclipse.che.util.CompilingGwtXmlGenerator</mainClass>
```

Comment:
question: why do we have exec-maven-plugin + classes instead of real maven plugin ?